Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2647,6 +2700,19 @@ public struct Moments<Scalar: TensorFlowFloatingPoint>: Differentiable {
 }
 
 public extension Tensor where Scalar: TensorFlowFloatingPoint {
+
+    /// Helper function that assess if `axis` is in the range `[-rank, rank)`, where `rank` is the rank of
+    /// the provided tensors.
+    @usableFromInline
+    internal func isAxisInRange(_ axis: Int) -> Bool {
```

Comment:
Please delete these duplicated versions of `isAxisInRange` and `areAxesInRange`. One declaration is sufficient.